### ISSUE TITLE
Compile submodules failing

### ISSUE THREAD

**Issue by Tactile-Taco** [Snoothy/UCR#214]
I can't run the initial build from power shell successfully. It fails on the compile submodules stage with the following message:

```
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\DataObjects\DTOs.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\ProviderInterface\ProviderInterface.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core\Core.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_Interception\Core_Interception.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\ProviderLogger\ProviderLogger.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Subscription Handling\SubscriptionHandler\SubscriptionHandler.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Subscription Handling\EmptyEventDictionary\EmptyEventDictionary.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Device Querying\DeviceLibrary\DeviceLibrary.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Device Querying\HidDeviceHelper\HidDeviceHelper.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Device Update Handling\PovHelper\PovHelper.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Subscription Handling\SubscriptionHandlingTests\Tests.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Provider Libraries\Device Update Handling\DeviceHandlers\DeviceHandlers.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_DS4WindowsApi\Core_DS4WindowsApi.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_Midi\Core_Midi.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_SpaceMouse\Core_SpaceMouse.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_TitanOne\Core_TitanOne.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_Tobii_Interaction\Core_Tobii_Interaction.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_ViGEm\Core_ViGEm.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Core_vJoyInterfaceWrap\Core_vJoyInterfaceWrap.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\Provider Unit Tests\Provider Unit Tests.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\SharpDX_DirectInput\SharpDX_DirectInput.csproj : error MSB4057: The target "Restore" does not exist in the project.
C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\Source\Core Providers\SharpDX_XInput\SharpDX_XInput.csproj : error MSB4057: The target "Restore" does not exist in the project.
Process 'msbuild.exe' exited with code 1.
   > "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\amd64\msbuild.exe" C:\Users\lahil\source\repos\UCR\submodules\IOWrapper\IOWrapper.sln /p:Configuration=Debug /target:Restore;Rebuild /verbosity:Quiet
   @ C:\Users\lahil\source\repos\UCR\submodules\IOWrapper

   at Nuke.Common.Tooling.ProcessExtensions.AssertZeroExitCode(IProcess process)
   at Nuke.Common.Tools.MSBuild.MSBuildTasks.MSBuild(MSBuildSettings toolSettings)
   at Build.<get_CompileSubmodules>b__33_1() in C:\Users\lahil\source\repos\UCR\build\Build.cs:line 83
   at Nuke.Common.Utilities.Collections.EnumerableExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action)
   at Nuke.Common.Execution.BuildExecutor.Execute(NukeBuild build, ExecutableTarget target, IReadOnlyCollection`1 previouslyExecutedTargets, Boolean failureMode)
   at Nuke.Common.Utilities.Collections.EnumerableExtensions.ForEach[T](IEnumerable`1 enumerable, Action`1 action)
   at Nuke.Common.Execution.BuildExecutor.Execute(NukeBuild build, IReadOnlyCollection`1 skippedTargets)
   at Nuke.Common.Execution.BuildManager.Execute[T](Expression`1 defaultTargetExpression)
```

**Comment by ShadyNawara** [comment]
this pull request seem to fix the issue #208

**Comment by Snoothy** [comment]
The develop branch has just been updated and AppVeyor works again. Try with the latest changes
